data/Ketankham.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Ketan Khamgaonkar",
+  "displayStatspublic":true,
+  "type": "personal",
+  "bio": "A research scholar turned no code developer, machine learning enthusiast and looking to make open source contributions",
+  "avatar": "https://github.com/Ketankham.png",\
```

Comment:
Following a recent update this is no longer needed:
```suggestion
```

Reply:
Yanked